package.json

```diff
@@ -4,7 +4,7 @@
   "description": "first",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "nodemon index.js"
```

Comment:
We should have `lint` and `lint:fix` scripts to, for everyone to be able to run lint and fix automatically

Reply:
```suggestion
    "start": "nodemon index.js",
    "lint:js": "eslint . --ext .js,.jsx",
    "lint:js:fix": "npm run lint:js -- --fix"
```